backend/code/archipelag/event_log/serializers.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework import serializers
+from archipelag.event_log.models import EventLog
+
+
+class EventLogSerializer(serializers.ModelSerializer):
+    owner = serializers.ReadOnlyField(source='owner.name')
```

Comment:
Better idea is add `owner_id` (`owner` is connected with field `owner_id`) and `owner_name`

Reply:
But what I can do whit this id? All I can is send from front end another request to get name. All I want to do to show owner name here

Reply:
Because in future when frontend need more informations about owner then frontend would be get owner data using owner ID